README.md

```diff
@@ -76,6 +76,92 @@ module "efs" {
   security_groups = ["sg-ac66e1d6"]
 }
 ```
+## monitoring
+Configure prometheus monitoring with alerts in opsgenie for an ECS cluster.
+The basic configuration includes the option to scrape for cloudwatch_metrics, and node-exporter metrics.
+It also allows to scrape for concourse metrics and to add any custom configuration.
+
+### Available variables:
+ * [`environment`]: String(required):  For what environment is this cluster needed (eg staging, production, ...)
+ * [`project`]: String(required): The project to tag Monitoring values
+ * [`cluster_name`]: String(required): The name of the ECS cluster.
+ * [`ecs_service_role`]: String(required): The name of the ECS IAM service role.
+ * [`desired_count`]: String(optional): The desired number of replicas of the ECS service for grafana and prometheus. (default: 1)
+ * [`prometheus_port`]: String(optional): The port where prometheus listens to (default: 9090)
+ * [`vpc_id`]: String(required): The VPC where the ECS cluster is
+ * [`ecs_sg`]: String(required):The ECS security group
+ * [`r53_zone`]: String(required): The DNS zone for prometheus and grafana
+ * [`r53_zone_prefix`]: String(optional): The DNS zone prefix for prometheus and grafana (default: "")
+ * [`prometheus_cpu`]: String(optional): The cpu dedicated to prometheus. (default: 0)
+ * [`prometheus_memory`]: String(optional): The memory dedicated to prometheus. (default: 100)
```

Comment:
Can you clarify the difference between `prometheus_memory` and `prometheus_memory_reservation`?